addons/mail/static/src/components/chat_window_header/chat_window_header.xml

```diff
@@ -33,6 +33,16 @@
                             <i class="fa fa-phone"/>
                         </div>
                     </t>
+                    <t t-if="chatWindow.thread and chatWindow.thread.hasInviteFeature and messaging.device.isMobile">
+                        <Popover popoverClass="'o_ThreadViewTopbar_invitePopover'" position="'bottom'">
+                            <button class="o_ThreadViewTopbar_inviteButton o_ThreadViewTopbar_button" title="Add users" t-on-click="chatWindow.onClickInviteButton">
+                                <i class="fa fa-lg fa-user-plus"/>
+                            </button>
+                            <t t-set="opened">
+                                <ChannelInvitationForm localId="chatWindow.threadView.channelInvitationForm.localId"/>
+                            </t>
+                        </Popover>
```

Comment:
I'm not fond of using `Popover`, especially for newer code.
There's a PR in progress that adds proper `Popover` in models (and component that is a messaging component and doesn't use slots). I would much prefer to use that.

Reply:
Popover is not really good in mobile indeed, better completely replace the thread view with the invite form in chat window (similar to what is done for the member list).